spec/draper/decorator_spec.rb

```diff
@@ -451,7 +451,7 @@ module Draper
       it "includes the context" do
         decorator = Decorator.new(double, context: {foo: "bar"})
 
-        expect(decorator.inspect).to include '@context={:foo=>"bar"}'
+        expect(decorator.inspect).to include "#{@context={:foo=>"bar"}}"
```

Comment:
Ruby 3.4 has changed the format of `Hash.inspect`. So I wrapped this with `#{}` to work both Ruby 3.4 and older.
Ref: https://bugs.ruby-lang.org/issues/20433